packages/console-sdk/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@mia-platform/console-sdk",
```

Comment:
We will not differ between frontend and backend sdk?

Reply:
i rename sdk for frontend specifically